test/integration/builder/inserts.js

```diff
@@ -1302,5 +1302,17 @@ module.exports = function(knex) {
           });
       });
     });
+
+    describe('#3553 insert with bigint', function() {
+      if (typeof BigInt === 'undefined') return;
```

Comment:
Hmm... does `"#3553 insert with bigint"` still get printed in the test case output?  If so, then we might want to move the conditional outside of the `describe(..)` block:

```
if(typeof BigInt !== 'undefined') {
  describe('#3553 insert with bigint', function() {
    //
  });
}
```

Reply:
I think this really not matters, because with node 8 drop this line will be removed